test/cli/test-bitswap.js

```diff
@@ -41,42 +39,32 @@ describe('bitswap', function () {
       httpAPI.stop(done)
     })
 
-    it('wantlist', (done) => {
+    it('wantlist', () => {
       const api = httpAPI.server.select('API')
 
       api.inject({
         method: 'GET',
         url: `/api/v0/block/get?arg=${key}`
-      }, (res) => {})
+      }, () => {})
 
-      nexpect.spawn('node', [process.cwd() + '/src/cli/bin.js', 'bitswap', 'wantlist'], {env})
-        .run((err, stdout, exitcode) => {
-          expect(err).to.not.exist
-          expect(exitcode).to.equal(0)
-          expect(stdout).to.be.eql([
-            key
-          ])
-          done()
-        })
+      return ipfs('bitswap wantlist').then((out) => {
+        expect(out).to.be.eql(key)
+      })
     })
 
-    it('stat', (done) => {
-      nexpect.spawn('node', [process.cwd() + '/src/cli/bin.js', 'bitswap', 'stat'], {env})
-        .run((err, stdout, exitcode) => {
-          expect(err).to.not.exist
-          expect(exitcode).to.equal(0)
-          expect(stdout).to.be.eql([
-            'bitswap status',
-            '  blocks received: 0',
-            '  dup blocks received: 0',
-            '  dup data received: 0B',
-            '  wantlist [1 keys]',
-            `    ${key}`,
-            '  partners [0]',
-            '    '
-          ])
-          done()
-        })
+    it('stat', () => {
+      return ipfs('bitswap stat').then((out) => {
```

Comment:
If we want the command to success it will, for when you want it to fail, there is `ipfs.fail('my cmd')` which expects failure and allows you to assert agains `stderr`